demos/python/sdk_wireless_camera_control/open_gopro/demos/gui/livestream.py

```diff
@@ -70,6 +70,7 @@ def parse_arguments() -> argparse.Namespace:
     parser.add_argument("ssid", type=str, help="WiFi SSID to connect to.")
     parser.add_argument("password", type=str, help="Password of WiFi SSID.")
     parser.add_argument("url", type=str, help="RTMP server URL to stream to.")
+    parser.add_argument("identifier", type=str, help="GoPro identifier.")

```

Comment:
This parameter already exists. It is added via the `add_cli_args_and_parse` below